sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/README.md

```diff
@@ -135,6 +133,43 @@ For more information on the OpenTelemetry project, please review the [OpenTeleme
 
 Refer to [`Program.cs`](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/tests/Azure.Monitor.OpenTelemetry.Exporter.Demo/Program.cs) for a complete demo.
 
+### Log Scopes
+
+Log [scopes](https://learn.microsoft.com/dotnet/core/extensions/logging#log-scopes) allow you to add additional properties to the logs generated by your application.
+Although the Azure Monitor Exporter does support scopes, this feature is off by default in OpenTelemetry.
+To leverage log scopes, you must explicitly enable them.
+
+To include the scope with your logs, set `OpenTelemetryLoggerOptions.IncludeScopes` to `true` in your application's configuration:
+```csharp
+var loggerFactory = LoggerFactory.Create(builder =>
+{
+    builder.AddOpenTelemetry(options =>
+    {
+        options.AddAzureMonitorLogExporter(o => o.ConnectionString = "InstrumentationKey=00000000-0000-0000-0000-000000000000");
```

Comment:
Shall we use a new API introduced in beta version?

Reply:
Are you referring to the `SDK.CreateLoggerProviderBuilder()`?
This API is still Experimental. I don't think we should include it in our Readme/docs until it ships and we can take it as a stable dependency.
https://github.com/open-telemetry/opentelemetry-dotnet/blob/36c586ddcc62cc1e4da82c6f9f65dba16b78161a/src/OpenTelemetry/Sdk.cs#L81